dt-core/admin/menu/tabs/tab-custom-fields.php

```diff
@@ -1538,6 +1540,15 @@ private function process_add_field( $post_submission ){
                     'customizable' => 'all',
                     'private' => $field_private,
                 ];
+            } elseif ( $field_type === 'boolean' ){
+                $new_field = [
+                    'name' => $post_submission['new_field_name'],
+                    'default' => [],
```

Comment:
@EthanW96 default here should probably be `false`

Reply:
Cool, I've adjusted this be marked as false by default.